pallets/runtime/tests/src/multisig.rs

```diff
@@ -99,7 +99,7 @@ fn join_multisig() {
         set_curr_did(Some(alice_did));
         assert_noop!(
             MultiSig::accept_multisig_signer_as_key(bob.clone(), bob_auth_id2),
-            Error::SignerAlreadyLinked
+            Error::SignerAlreadyLinkedToMultisig
```

Comment:
If there was only one test for the old error, then we haven't been testing all three failure cases.

Please add a test case for the other two failure cases.